<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QMainWindow" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
        QFrame { background-color: #1e1e1e; }
        QPushButton { 
            background: none; 
            color: #eee; 
            font-size: 16px; 
            padding: 12px 10px; 
            border-radius: 5px;
            text-align: left;
        }
        QPushButton:hover { background-color: #2c2c2c; }
        QPushButton:checked { background-color: #3a3a3a; }
       </string>
      </property>
      <layout class="QVBoxLayout" name="sidebarLayout">
       <item>
        <widget class="QLabel" name="adminTitle">
         <property name="styleSheet">
          <string notr="true">font-size:20px; font-weight:bold; color:#fff;</string>
         </property>
         <property name="text">
          <string>ðŸŽ§ Admin Dashboard</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerDashboard">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="welcomeLabel">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:20px; font-weight:bold; color:#fff;</string>
         </property>
         <property name="text">
          <string>Welcome back!, {Username}!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerWelcome">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="requestsBtn">
         <property name="text">
          <string>Pending Artist Requests</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manageBtn">
         <property name="text">
          <string>Artists Management</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="usersBtn">
         <property name="text">
          <string>Users Management</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="analyticsBtn">
         <property name="text">
          <string>Analytics</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerMiddle">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="profileBtn">
         <property name="text">
          <string>Profile</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logoutBtn">
         <property name="styleSheet">
          <string notr="true">background:red; color:white; font-weight:bold; border-radius:5px; padding:12px;</string>
         </property>
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="artistRequestsPage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <widget class="QLabel" name="artistRequestsTitle">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>11</y>
          <width>264</width>
          <height>27</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:22px; font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Pending Artist Requests</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QTableWidget" name="artistRequestsList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>911</width>
          <height>711</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>60</y>
          <width>601</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:18px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:18px;</string>
        </property>
        <property name="text">
         <string>Enter Artist ID here:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>60</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:19px;</string>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="artistsRequestTable">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>891</width>
          <height>641</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#FFFFFF ; color: #000000;              border: 10px; solid #FFFFFF;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Artist ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Stage Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Real First Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Real Last Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Details</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="">
        <layout class="QHBoxLayout"/>
       </widget>
      </widget>
      <widget class="QWidget" name="manageArtistsPage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="manageArtistsTitle">
          <property name="styleSheet">
           <string notr="true">font-size:22px; font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Artist Management</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="allArtistsList"/>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="removeArtistBtn">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="banArtistBtn">
            <property name="text">
             <string>Ban</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="analyticsPage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="analyticsTitle">
          <property name="styleSheet">
           <string notr="true">font-size:22px; font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Analytics</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="analyticsList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="usersPage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="usersTitle">
          <property name="styleSheet">
           <string notr="true">font-size:22px; font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Users Management</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="searchUserInput">
            <property name="placeholderText">
             <string>Search user by name...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchUserBtn">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="usersList"/>
        </item>
        <item>
         <widget class="QPushButton" name="deleteUserBtn">
          <property name="text">
           <string>Delete User</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="profilePage">
       <property name="styleSheet">
        <string notr="true">background-color: #121212; color: #fff;</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="profileLabel">
          <property name="styleSheet">
           <string notr="true">font-size:22px; font-weight:bold;</string>
          </property>
          <property name="text">
           <string>ðŸ‘¤ Admin Profile</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="themeToggleBtn">
          <property name="text">
           <string>Switch Light/Dark Theme</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
